packages/terra-base/src/Base.jsx

```diff
@@ -73,20 +75,25 @@ class Base extends React.Component {
       children,
       locale,
       customMessages,
+      strictMode,
       translationsLoadingPlaceholder,
       ...customProps
     } = this.props;
 
     const messages = Object.assign({}, this.state.messages, customMessages);
+    const renderStrictMode = strictMode ? (<React.StrictMode>{children}</React.StrictMode>) : children;
```

Comment:
nit: would it make more sense to name this variable `renderChildren`, since we won't always be rendering in strict mode?

Reply:
Updated: https://github.com/cerner/terra-core/pull/1890/commits/90c983b9fb7a4492f6805d111c31f8ba02d91a61